src/index.html

```diff
@@ -5,9 +5,60 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
+    <link rel="stylesheet" href="reset.css">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars-form">
+      <div class="stars stars--0">
+        <div class="stars__star">
+          
+        </div>
```

Comment:
instead of `div` you can use inline tag like `span` 
```suggestion
        <span class="stars__star"></span>
```